src/api/zodClient.ts

```diff
@@ -3310,6 +3326,49 @@ const endpoints = makeApi([
             },
         ],
     },
+    {
+        method: 'put',
+        path: '/v1/projects/:project/features/:feature/overrides/current',
+        alias: 'OverridesController_updateFeatureOverride',
+        requestFormat: 'json',
+        parameters: [
+            {
+                name: 'project',
+                type: 'Path',
+                schema: z.string(),
+            },
+            {
+                name: 'feature',
+                type: 'Path',
+                schema: z.string(),
+            },
+            {
+                name: 'body',
+                type: 'Body',
+                schema: UpdateUserOverrideDto,
+            }
+        ],
+        response: Override,
+        errors: [
+            {
+                status: 401,
+                schema: z.void(),
+            },
+            {
+                status: 403,
+                schema: z.void(),
+            },
+            {
+                status: 404,
+                schema: NotFoundErrorResponse,
+            },
+            {
+                status: 409,
```

Comment:
is a 409 conflict possible when updating an override?

Reply:
it does not.. removed it